python-frontend/src/main/java/org/sonar/python/types/InferredTypes.java

```diff
@@ -156,8 +156,10 @@ public static InferredType fromTypeshedProtobuf(SymbolsProtos.Type type) {
         String typeName = type.getFullyQualifiedName();
         return typeName.isEmpty() ? anyType() : runtimeType(TypeShed.symbolWithFQN(typeName));
       case TYPE_ALIAS:
-      case CALLABLE:
         return fromTypeshedProtobuf(type.getArgs(0));
+      case CALLABLE:
+        // this should be handled as a function type
```

Comment:
Do we want a ticket for that? Are there FNs we would be able to avoid if we tracked such type correctly?

Reply:
Good point, I'll create a ticket for this